web/templates/compose.hbs

```diff
@@ -13,11 +13,12 @@
     <div id="compose_controls" class="new-style">
         <div id="compose_buttons">
             <div class="new_message_button reply_button_container">
-                <button type="button" class="button small compose_reply_button"
-                  id="left_bar_compose_reply_button_big"
-                  data-tooltip-template-id="compose_reply_message_button_tooltip_template">
-                    {{t 'Compose message' }}
-                </button>
+                <span class="compose_reply_button_wrapper" data-tooltip-template-id="compose_reply_message_button_tooltip_template">
```

Comment:
The commit message mentions a div but this is a span. I think a `<div>` is a better choice here, if we need a wrapper element, just so it's hinted in the source code that this thing doesn't actually display as an inline element the way a `<span>` does.

Reply:
Good catch, and this is indeed a mistake from my side. Fixed, thanks!